### PR TITLE
Create Schema Relationships

### PR DESCRIPTION
**Pull request by ryanmahan**
All incident relationships are one to many. Council from legal_case are one to one. Looks like `black` might have changed some of the quotes from single to double quotes in the scraper.

Entity relationship diagram of the relationships defined in this PR:

PDT ER

### CODE REVIEW

**Comment by JMStudiosJoe** [comment]
Looks clean from high level but can someone with better pythons eyes have a look? @DMalone87 @dwreeves

File: backend/database/models/tag.py [hunk 1]
```diff
@@ -3,4 +3,7 @@
 
 class Tag(db.Model):
     id = db.Column(db.Integer, primary_key=True)
+    incident_id = db.Column(
```
Comment (DMalone87) [inline backend/database/models/tag.py:6]: This one could probably be a many to many... Thoughts?

File: backend/database/models/incident.py [hunk 2]
```diff
@@ -74,60 +73,47 @@ class Incident(db.Model):
...
-    # (Note) Removed all the `description_*` prefixes.
+    victims = db.relationship("Victim", backref="incident")
+    descriptions = db.relationship("Description", backref="incident")
+    tags = db.relationship("Tag", backref="incident")
+    participants = db.relationship("Particpant", backref="incident")
+    multimedias = db.relationship("Multimedia", backref="incident")
+    investigations = db.relationship("Investigation", backref="incident")
+    results_of_stop = db.relationship("ResultOfStop", backref="incident")
+    actions = db.relationship("Action", backref="incident")
+    use_of_force = db.relationship("UseOfForce", backref="incident")
```
Comment (DMalone87) [inline backend/database/models/incident.py:84]: Heads up @jcockhren

File: backend/database/models/tag.py (hunk 1 above)
Comment (DMalone87) [reply backend/database/models/tag.py:6]: @dwreeves? @jcockhren?

**Review by DMalone87 — APPROVED** [review]

**Comment by JMStudiosJoe** [comment]
once conflicts resolved then will merge

**Comment by ryanmahan** [comment]
Ready for merge now. Made some updates with @dwreeves to standardize table names and fix a few bugs here and there

File: backend/database/models/incident.py [hunk 3]
```diff
@@ -86,58 +113,24 @@ class Incident(db.Model, CrudMixin):
...
+class UseOfForce(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    incident_id = db.Column(db.Integer, db.ForeignKey("incident.id"))
+    item = db.Text()
+
+
+class ResultOfStop(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    incident_id = db.Column(db.Integer, db.ForeignKey("incident.id"))
+    result = db.Column(db.Text)
```
Comment (ryanmahan) [inline backend/database/models/incident.py:136]: @dwreeves should each of these get its own file? They're all one to many with incident

File: backend/database/models/tag.py (hunk 1 above)
Comment (dwreeves) [reply backend/database/models/tag.py:6]: I'm not sure. I feel like this should be an array of strings and not even be a table. This is how the Berkeley Copwatch data model had it, though. They had more attributes in their tags table too, I think.

File: backend/database/models/incident.py (hunk 3 above)
Comment (dwreeves) [reply backend/database/models/incident.py:136]: oh good catch. Uhhh.... probably.

**Comment by dwreeves** [comment]
LGTM.

We have a few things we're not sure of yet, but I don't think they're a big deal to resolve now; we can solve them here but they're not a blocker for merge IMO.

**Comment by JMStudiosJoe** [comment]
is this ready to be merged?

**Comment by ryanmahan** [comment]
@JMStudiosJoe yes!

**Pull request merged**